tests/wpt/mozilla/tests/mozilla/interfaces.worker.js

```diff
@@ -82,6 +83,7 @@ test_interfaces([
   "WebGLShaderPrecisionFormat",
   "WebGLTexture",
   "WebGLUniformLocation",
+  "WebKitCSSMatrix",
```

Comment:
These changes are due to the LegacyWindowAlias attribute present in the relevant WebIDL files. I have a subsequent PR which makes further changes to global enumeration for non-window globals which should make them go away again, but I didn't want to include both changes in this PR.